src/pages/PrivacyStatement.tsx

```diff
@@ -3,713 +3,220 @@ import styles from "./ImprintAndPrivacy.module.scss";
-              </li>
-              <li>
-                <a className="index-link" href="#mOverview">
-                  Übersicht der Verarbeitungen
-                </a>
-              </li>
-              <li>
-                <a className="index-link" href="#m13">
-                  Maßgebliche Rechtsgrundlagen
-                </a>
-              </li>
-              <li>
-                <a className="index-link" href="#m27">
-                  Sicherheitsmaßnahmen
-                </a>
-              </li>
-              <li>
-                <a className="index-link" href="#m25">
-                  Übermittlung von personenbezogenen Daten
-                </a>
-              </li>
-              <li>
-                <a className="index-link" href="#m12">
-                  Löschung von Daten
-                </a>
-              </li>
-              <li>
-                <a className="index-link" href="#m225">
-                  Bereitstellung des Onlineangebotes und Webhosting
-                </a>
-              </li>
-              <li>
-                <a className="index-link" href="#m182">
-                  Kontakt- und Anfragenverwaltung
-                </a>
-              </li>
-              <li>
-                <a className="index-link" href="#m17">
-                  Newsletter und elektronische Benachrichtigungen
-                </a>
-              </li>
-              <li>
-                <a className="index-link" href="#m408">
-                  Umfragen und Befragungen
-                </a>
-              </li>
-              <li>
-                <a className="index-link" href="#m10">
-                  Rechte der betroffenen Personen
-                </a>
-              </li>
-              <li>
-                <a className="index-link" href="#m42">
-                  Begriffsdefinitionen
-                </a>
-              </li>
+          <LegalSection header={t("privacy.tableOfContents")}>
+            <ul>
+              <li><a href="#mIntro"> {t("privacy.intro.header")} </a></li>
+              <li><a href="#mResponsible"> {t("privacy.responsibilities.header")} </a></li>
+              <li><a href="#mOverview"> {t("privacy.overviewOverProcesses.header")} </a></li>
+              <li><a href="#mLegalBases"> {t("privacy.relevantLegalBases.header")} </a></li>
+              <li><a href="#mSecurity"> {t("privacy.security.header")} </a></li>
+              <li><a href="#mDeletion"> {t("privacy.deletionOfData.header")} </a></li>
+              <li><a href="#mCookies"> {t("privacy.useOfCookies.header")} </a></li>
+              <li><a href="#mWebhosting"> {t("privacy.webhosting.header")} </a></li>
+              <li><a  href="#mVote"> {t("privacy.votes.header")} </a></li>
+              <li><a href="#mChanges"> {t("privacy.updates.header")} </a></li>
+              <li><a href="#mRights"> {t("privacy.rights.header")} </a></li>
+              <li><a href="#mDefinitions"> {t("privacy.definitions.header")} </a></li>
```

Comment:
why `m`?

Reply:
Also, I think it is uncommon to have any upper cased letters in urls as those fragment links are case sensitive.

Reply:
Yeah I agree. I forgot that it is an url

Reply:
> I think it is uncommon to have any upper cased letters in urls as those fragment links are case sensitive.

The issue is that URL fragments are case *in*sensitive in Internet Explorer (but they aren't in other browsers).